### PR TITLE
Add PoolFormer support in exporters.onnx

### PR DESCRIPTION
**Pull request by BakingBrains**
This PR tries to add ```PoolFormer``` models for ONNX support

### CODE REVIEW

**Comment by BakingBrains** [comment]
Tested with
```python -m optimum.exporters.onnx --model sail/poolformer_s12 --task image-classification out_poolformer```

poolonnx

**Comment by BakingBrains** [comment]
Also tested ONNX model with

```
from transformers import AutoFeatureExtractor
from optimum.onnxruntime import ORTModelForImageClassification
from optimum.pipelines import pipeline

feature_extractor = AutoFeatureExtractor.from_pretrained("optimum\out_poolformer")

# Loading already converted and optimized ORT checkpoint for inference
model = ORTModelForImageClassification.from_pretrained("optimum\out_poolformer")

onnx_img_classif = pipeline(
    "image-classification", model=model, feature_extractor=feature_extractor
)

image_path = "cat.jpg"

pred = onnx_img_classif(image_path)
print("Top-5 predicted classes:", pred)
```

Output:
```
Top-5 predicted classes: [{'score': 0.17694829404354095, 'label': 'Siamese cat, Siamese'}, {'score': 0.09261114150285721, 'label': 'Egyptian cat'}, {'score': 0.08993127942085266, 'label': 'lynx, catamount'}, {'score': 0.05380868911743164, 'label': 'weasel'}, {'score': 0.02326379530131817, 'label': 'tiger cat'}]
```

File: tests/exporters/exporters_utils.py [hunk 1]
```diff
@@ -74,6 +74,7 @@
         "hf-internal-testing/tiny-random-vision_perceiver_conv": ["image-classification"],
     },
     # "rembert": "google/rembert",
+    "poolformer": "sail/poolformer_s12",
```
Comment (fxmarty) [inline tests/exporters/exporters_utils.py:77]:
```suggestion
    "poolformer": "hf-internal-testing/tiny-random-PoolFormerModel",
```

Can you try with this one (sometimes the models from hf-internal-testing that were autogenerated are not good) to keep the tested models small?

**Review by fxmarty — APPROVED** [review]
Thank you for the PR and the test! I just left a comment for the model used at testing but LGTM

File: tests/exporters/exporters_utils.py (hunk 1 above)
Comment (BakingBrains) [reply tests/exporters/exporters_utils.py:77]: @fxmarty sure I will test.

Comment (fxmarty) [reply tests/exporters/exporters_utils.py:77]: if you want you can just "commit suggestion" and I will run the workflows.

Comment (BakingBrains) [reply tests/exporters/exporters_utils.py:77]: @fxmarty Just updated
22ss

**Comment by BakingBrains** [comment]
@fxmarty done

**Comment by fxmarty** [comment]
Thank you for your contribution!

**Pull request merged**